app/views/camp/insert.php

```diff
@@ -0,0 +1,264 @@
+                    Assets
+                </a>
+            </li>
+
+            <li>
+                <a href="#">
+                    Requests
+                </a>
+            </li>
+
+        </ul>
+    </div>
+    <div id="page-content-wrapper" style="background-color: #ffffff;">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-12">
+                    <div class="page-header">
+                        <h1 style="text-align: center;">
+                            Register Camp<br/>
+                        </h1>
+                        <br /><br />
+
+                        <?php
+                        if ($is_success !== null) {
+                            if ($is_success == true) {
+                                echo "<div class='alert alert-success'>Camp registered</div>";
+                            }
+                            else {
+                                echo "<div class='alert alert-danger'>Camp not registered</div>";
+                            }
+                            echo "<br />";
+                        }
+                        ?>
+
+                        <form action='' method='POST'>
+                            <fieldset>
+                                <div class="panel panel-primary">
+                                    <div class="panel-heading" style="font-size: large; padding-bottom: 15px">
+                                        <b>Camp Details</b>
+                                        <div class="btn btn-info" id="show" style="float: right">Hide</div>
+                                    </div>
+
+                                    <div class="panel-body" id="panelBody">
+                                        <label for="name">Camp Name </label>
+                                        <input class="form-control" placeholder="Enter Camp Name" name='name' required/><br>
+                                        <label for="organisation_id">Organisation </label>
+                                        <input class="hidden" name='organisation_id' id='organisation_id' required/><br>
+                                        <input class="form-control" placeholder="Select Organisation" name='organisation' id='organisation'/><br><br>
+                                        <label for="population">Population: </label>
+                                        <input class="form-control" placeholder="Enter population" type="text" name="population" required/><br>
+                                        <label for="volunteers">Volunteers: </label>
+                                        <input class="form-control" type="text" placeholder="Enter volunteers" name="volunteers" required/><br>
+                                        <label for="status">Status: </label>
+                                        <input class="form-control" type="text" placeholder="Enter status" name="status" required/><br>
```

Comment:
Camps should provide a rough estimate initially and it can be changed (using edit)
[to be implemented] afterwards if conditions of disaster change.